recipes/bibmon/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bibmon" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bibmon-{{ version }}.tar.gz
+  sha256: d2cd59ca1b1b32224e03a3a610f0ced3d15bb3f459835d652c147ecf12439529
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
No need to change here, as stated in the comment https://github.com/conda-forge/staged-recipes/pull/27400#issuecomment-2314430863

Reply:
This is a required change and is unrelated to that comment. This is the script line we use in conda-forge to ensure we won't get accidental wheels into the mix.

Reply:
I'm sorry for the misunderstanding. Done.